src/sidebar/util/test/annotation-sharing-test.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const sharingUtil = require('../annotation-sharing');
+
+describe('sidebar.util.annotation-sharing', () => {
```

Comment:
Can you change this to `sidebar/util/annotation-sharing` per our preferred conventions.